.github/workflows/deployDev.yml

```diff
@@ -94,3 +94,41 @@ jobs:
           azure_creds: ${{ secrets.AZURE_CREDENTIALS }}
           client_tarball: client.tgz
           deploy_env: ${{ inputs.deploy_env }}
+
+  smoke_test_front_and_back_end:
```

Comment:
Do we want to `always()` run this check? As it is, if something before it fails, this will be canceled or skipped.

Reply:
Do you think we should allow this to be canceled or skipped during a failed dev deploy? Related to your other comment I can see the value in always running the smoke test on prod or even staging even if those deploy workflows failed, but not sure if it would be needed for dev